src/goto-instrument/replace_function_bodies.h

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************\
+
+Module: Replace bodies of goto functions
+
+Author: DiffBlue Limited.
+
+\*******************************************************************/
+
+#ifndef CPROVER_GOTO_INSTRUMENT_REPLACE_FUNCTION_BODIES_H
+#define CPROVER_GOTO_INSTRUMENT_REPLACE_FUNCTION_BODIES_H
+
+#include <regex>
+#include <vector>
+#include <iostream>
+
+#include <goto-programs/goto_functions.h>
+#include <util/std_code.h>
+#include <util/std_types.h>
+#include <util/message.h>
+
+void replace_function_bodies(
+  goto_functionst &functions,
+  symbol_tablet &symbol_table,
+  const std::regex &functions_regex,
+  bool only_generate,
+  const std::string &replace_options,
+  messaget &messages);
+
+#define OPT_REPLACE_FUNCTION_BODY "(replace-function-body):"
+#define OPT_GENERATE_FUNCTION_BODY "(generate-function-body):"
+#define OPT_REPLACE_FUNCTION_BODY_OPTIONS "(replace-function-body-options):"
+
+#define HELP_REPLACE_FUNCTION_BODY                                             \
+  " --replace-function-body <regex>\n"                                         \
+  "                              Replace bodies of function matching regex\n"
+
+#define HELP_GENERATE_FUNCTION_BODY                                            \
+  " --generate-function-body <regex>\n"                                        \
+  "                              Like replace-function-body, but ignore "      \
```

Comment:
Dangling reference to `replace-function-body` if `HELP_REPLACE_FUNCTION_BODY` is not included.